DepositeCalcTests/Tests/RegisterPageTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using DepositeCalcTests.Pages;
+using DepositeCalcTests.Utilities;
+using NUnit.Framework;
+
+namespace DepositeCalcTests.Tests
+{
+    internal class RegisterPageTests : BaseTest
+    {
+        private RegisterPage registerPage;
+        private ApiHelper apiHelper;
```

Comment:
1. `apiHelper` object is used only in the `Setup` method, so you don't need to declare it global.
2. Usually the helper classes are static, and you just call its methods directly: ApiHelper.Delete()
```suggestion
```